djangocms_alias/admin.py

```diff
@@ -111,12 +112,17 @@ class AliasContentAdmin(*alias_content_admin_classes):
     actions = None
     change_form_template = "admin/djangocms_alias/aliascontent/change_form.html"
 
+    def get_queryset(self, request):
+        queryset = super().get_queryset(request)
+        queryset = queryset.annotate(alias_category_translations_ordered=Lower("alias__category__translations__name"))
```

Comment:
```suggestion
        # Force the category set to Lower to be able to sort the category in 
        # asc / desc order case insensitive 
        queryset = queryset.annotate(alias_category_translations_ordered=Lower("alias__category__translations__name"))
```

A comment here to let others know what this is doing.